org.goldcoin.goldcoin-qt.json

```diff
@@ -0,0 +1,67 @@
+{
+  "id": "org.goldcoin.goldcoin-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "20.08",
+  "command": "goldcoin-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "goldcoin",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/goldcoin-qt"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
+          "url": "https://github.com/goldcoin/goldcoin/releases/download/v0.14.9.1/goldcoin-0.14.9.1-x86_64-linux-gnu.tar.gz",
+          "sha256": "a9c6c6d6fdd9f021c1d8e22a18d4c83199f36362bf1579839f14af7c5c456204"
+        }
+      ]
+    },
+    {
+      "name": "misc",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dp -m 644 goldcoin-icon-transparent.png /app/share/icons/hicolor/scalable/apps/org.goldcoin.goldcoin-qt.png",
+        "install -Dp -m 644 goldcoin-qt.desktop /app/share/applications/org.goldcoin.goldcoin-qt.desktop",
+        "desktop-file-edit --set-key=Icon --set-value=org.goldcoin.goldcoin-qt.png /app/share/applications/org.goldcoin.goldcoin-qt.desktop",
```

Comment:
icon is NOT a filename. (aka no `.png` at the end)

Reply:
I will replace with .svg

Reply:
that's not what I meant. It should be the icon *name*. ie no extension.

Reply:
It works great as it is. Is this about flatpak requirement?

Reply:
If it works, it does only coincidentally. It should be ID without the extension.